src/Telegram.php

```diff
@@ -904,17 +904,24 @@ public function runCommands($commands)
         $this->run_commands  = true;
         $this->botan_enabled = false;   // Force disable Botan.io integration, we don't want to track self-executed commands!
 
-        $result = Request::getMe()->getResult();
+        $result = Request::getMe();
 
-        if (!$result->getId()) {
-            throw new TelegramException('Received empty/invalid getMe result!');
+        if ($result->isOk()) {
+            $result = $result->getResult();
+
+            $bot_id       = $result->getId();
+            $bot_name     = $result->getFirstName();
+            $bot_username = $result->getUsername();
+        } else {
+            $bot_id       = $this->getBotId();
+            $bot_name     = $this->getBotUsername();
+            $bot_username = $this->getBotUsername();
         }
 
-        $bot_id       = $result->getId();
-        $bot_name     = $result->getFirstName();
-        $bot_username = $result->getUsername();
+        if (!$this->isAdmin($bot_id)) {
```

Comment:
I don't think this is an issue here, since it works in first loop then starts failing, isnt there a check in addadmin to return false when Id is already admin? This is causing this.